.scala-steward.conf

```diff
@@ -2,10 +2,8 @@
 # Default config settings are here: https://github.com/scala-steward-org/scala-steward/blob/c1aa49d38b3d5fa6adf63f59de240680d7f9135b/docs/repo-specific-configuration.md
 # Check if department-wide Scala Steward GitHub Action has broken on a team’s repo here: https://github.com/guardian/scala-steward-public-repos/actions/workflows/public-repos-scala-steward.yml
 
-# 10am Monday to Friday PR cron
-pullRequests.frequency = "0 10 * * 1-5"
+pullRequests.frequency = "7 days"
```

Comment:
Given this project only has a handful of dependencies - effectively Play and two AWS libraries, it might be worth making this even less frequent. I'd be happy with every 30 days